app/controllers/people_controller.rb

```diff
@@ -0,0 +1,42 @@
+class PeopleController < ApplicationController
+	def index
+		@people = Person.all
+	end
+
+	def show
+		@person = Person.find(params[:id])
+		@completed_tasks = @person.completed_tasks
+	end
+
+	def tasks
+		@person = Person.find(params[:id])
+		@working_tasks = @person.working_tasks
+		@completed_tasks = @person.completed_tasks
```

Comment:
Do you need this additional instance variables or could you use the `@person` variable to retrieve this info in your view?